src/state/quick-bar-mixin.ts

```diff
@@ -36,20 +37,42 @@ export default <T extends Constructor<HassElement>>(superClass: T) =>
     }
 
     private _showQuickBar(e: KeyboardEvent, commandMode = false) {
-      if (
-        !this.hass?.user?.is_admin ||
-        !this.hass.enableShortcuts ||
-        this._inInputField(e)
-      ) {
+      if (!this._canShowQuickBar(e)) {
         return;
       }
 
       showQuickBar(this, { commandMode });
     }
 
-    private _inInputField(e: KeyboardEvent) {
-      return ["INPUT", "TEXTAREA"].includes(
-        (e.composedPath()[0] as HTMLElement).tagName
+    private _canShowQuickBar(e: KeyboardEvent) {
+      return (
+        this.hass?.user?.is_admin &&
+        this.hass.enableShortcuts &&
+        this._canOverrideAlphanumericInput(e) &&
+        !isMobileOrTablet()
```

Comment:
Is this just for codemirror? Can't we just detect it was fired in a codemirror element instead of blocking all mobiles and tablets?

I think a tablet with a Bluetooth keyboard should just work with quickbar, codemirror has some workarounds for iOS, which probably causes this problem.

I think it is safe to just filter out `div.CodeMirror-code`, we could even do that in the `ha-code-editor` element instead of here.

Reply:
Adding `this.addEventListener("keydown", (ev) => ev.stopPropagation());` to `ha-code-editor`

Reply:
You don't think using a class name, and of a 3rd party tool, is going to be too brittle?

Reply:
Went ahead and did the change and pushed. Keeping the "`isMobileOrTablet`" helper though as I suspect it will be useful as we work out mobile shortcut/gesture strategies.

Reply:
Oh I didn't see your 2nd comment :P Okay I'll add there.

Reply:
Done